Rakefile

```diff
@@ -48,8 +167,69 @@ namespace :test do
     end
     Dir.chdir( original_dir )
   end
+
+  desc <<-EOS
+Run the base beaker acceptance tests
+#{USAGE}
+  EOS
+  task :base  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/base'})
+  end
+
+  desc <<-EOS
+Run the hypervisor beaker acceptance tests
+#{USAGE}
+  EOS
+  task :hypervisor  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/hypervisor'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a pe install.
+#{USAGE}
+  EOS
+  task :puppetpe  => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/pe/install.rb', :tests => 'acceptance/tests/puppet'})
```

Comment:
@er0ck  - it seems that because of BKR-670, the options we have written to merged_options.rb are ignored when running in preserved_hosts_mode, so the :pre-suite we have defined here is ignored anyway when a hosts file is specified. Would have to set ENV['TESTS'] and/or ENV['OPTIONS'] for any additional options. Unless I am missing an easier/better way to do this.